ee/desktop/menu/menu_template.go

```diff
@@ -17,6 +17,7 @@ type TemplateData struct {
 	LauncherRevision string `json:",omitempty"`
 	GoVersion        string `json:",omitempty"`
 	ServerHostname   string `json:",omitempty"`
+	LastUpdateTime   int64  `json:",omitempty"`
```

Comment:
I think I want to be clear about what this means. 

It's not the time k2 generates the data. It's it's the time we last noticed new data and wrote it to disk?

Reply:
Correct, including if someone were to open the menu template file locally and modify it.

Reply:
Hrm.... If K2 starts including the time it was generated k2 side, does that change how we want to name things?
`LastServerGeneratedTime` and `LastUpdateTime`? Something else?

Reply:
`LastServerGeneratedTime` and `LastMenuUpdateTime`?

Reply:
Sure

Reply:
Actually, `LastServerGeneratedTime` would just come directtly from k2 - so no need for a launcher template variable.

Reply:
Agreed. I'm thinking about the names and what to call things.

Reply:
Renamed `LastUpdateTime` to `LastMenuUpdateTime`